YAFC/Workspace/ProductionTable/ProductionTableFlatHierarchy.cs

```diff
@@ -105,6 +115,17 @@ public class FlatHierarchy<TRow, TGroup> where TRow : ModelObject<TGroup>, IGrou
             for (int i = 0; i < flatRecipes.Count; i++) {
                 var recipe = flatRecipes[i];
                 var item = flatGroups[i];
+
+                nextRowIsHighlighted = typeof(TRow) == typeof(RecipeRow) && rowHighlighting[i] != RowHighlighting.None;
+                if (nextRowIsHighlighted) {
+                    nextRowBackgroundColor = GetHighlightingBackgroundColor(rowHighlighting[i]);
+                    nextRowTextColor = GetHighlightingTextColor(rowHighlighting[i]);
+                }
+                else {
+                    nextRowBackgroundColor = bgColor;
+                    nextRowTextColor = SchemeColor.BackgroundText;
```

Comment:
Good idea.

But perhaps this could be part of a refactoring effort that would require a follow-up PR. @randblade has been patiently awaiting this feature, and I don't want them to wait any longer ;-)

Reply:
I am a bit on the fence: I don't really like adding 'legacy'... But I also agree to not let @RandBlade wait unnecessary.

@have-fun-was-taken are you planning to make a new release soon? If not, @RandBlade needs to wait nonetheless... (unless they are able to run YAFC-CE from code)

Reply:
This project is devs-oriented. In other words, it prioritizes actions that increase the expected development power of the team. Whether the refactor attracts more dev power, or merging the PR will keep @shihan42 motivated to contribute more, can be a lengthy argument full of hypotheticals, so I leave this decision up to @shihan42. The only thing I ask is to make an issue about the refactor if he decides to keep it as is.

Reply:
Ok, I'll split the work. For now, we'll proceed with this pull request and its legacy code. However, I've already opened an issue regarding the necessary refactoring. See #91